RevenueCatUI/Templates/Template4View.swift

```diff
@@ -284,10 +284,10 @@ private struct PackageButton: View {
                     .frame(maxHeight: .infinity)
             }
             .background { // Discount overlay
-                if let discount = self.package.discountRelativeToMostExpensivePerMonth {
-                    self.discountOverlay(discount)
+                if let badge = self.package.localization.offerBadge, !badge.isEmpty {
```

Comment:
This is unrelated to the changes in this PR, but I think this would be a lot more readable if we only used one `.background` modifier and then rendered the entire background in one subview extracted out to a local `@ViewBuilder` function